### PR TITLE
Project

### PR DESCRIPTION
**Pull request by rbhatt1999**
# Hello Code Reviewer 👋 👋 
image

## In this pull request following changes have been made:
- Added the splash screen, which refers to my home page.
- Added the Signup and log-in pages.
- Added Home page (categories page).
- Added Transactions page.
- Added the "Add a new category" form page.
- Added the "Add a new transaction" form page.
- Added unit test using RSpec.
- Ensured the database table matches the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png) diagram.
- Ensured there was no linter error.
- Deployed the app using [render](https://budget-6lkf.onrender.com/).
- Updated my readme file.

### This part of my project features the complete project version of my budget app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,183 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-6lkf.onrender.com/)- Powered by Render
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Presentation <a name="presentation"></a>
+
+- [Video Presentation Link](https://www.loom.com/share/a57474cdf4d04160b19374afad20f577)- Powered by Loom
```
Comment (brytebee) [inline README.md]: Great presentation video 👏 

One thing is left though, you are required to show a piece of code you're proud of, explain it and tell us why you are proud of it.

File: README.md [hunk 2]
```diff
@@ -1 +1,183 @@
...
+### Key Features <a name="key-features"></a>
+
+> - Categorized lists.
+> - Navigation buttons for moving between pages.
+> - Ability to add new categories with mandatory name and icon fields.
+> - Ability to add new transactions with mandatory name, amount, and category fields.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-6lkf.onrender.com/)- Powered by Render
```
Comment (brytebee) [inline README.md:55]: 👏👏👏

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @rbhatt1999 👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️
- Great use of devise and cancan gems for auth.✔️
- All tests pass and are green.✔️

##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (rbhatt1999) [reply README.md:55]: is there any issue in live demo? @brytebee

File: README.md (hunk 1 above)
Comment (rbhatt1999) [reply README.md]: https://www.loom.com/share/63bc64cde192458badbb6da120743994

File: README.md (hunk 2 above)
Comment (prolajumokeoni) [reply README.md:55]: Hi, no he was just highlighting the great work you put in 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @rbhatt1999 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 👍🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**